runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1267,6 +1267,13 @@ TR_VectorAPIExpansion::generateAddressNode(TR::Compilation *comp, TR::Node *arra
    return aladdNode;
    }
 
+TR::Node *
+TR_VectorAPIExpansion::generateAddressNode(TR::Compilation *comp, TR::Node *base, TR::Node *offset)
```

Comment:
The compilation object is not used in this function.  Does it need to be part of the API?

Reply:
Right. I will remove it.

Reply:
Done.